src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,39 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 100;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MAX_PASSWORD_LENGTH = 20;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (!(user.getLogin() != null && storageDao.get(user.getLogin()) == null)) {
```

Comment:
You can check login for null before checking that user exists
```suggestion
        if (user.getLogin() == null) {
            throw new RuntimeException("Enter user's login");
        }
        if (storageDao.get(user.getLogin()) != null) {
```